OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/WorkshopServiceTests.cs

```diff
@@ -347,6 +347,28 @@ public async Task Update_WhenEntityIsValid_ShouldReturnUpdatedEntity([Random(2,
         result.Teachers.Should().BeEquivalentTo(expectedTeachers);
     }
 
+    [Test]
+    public async Task Update_WhenEntityIsValidAndAvaliableSeatsZero_ShouldReturnUpdatedEntityWithMaxUintAvaliableSeats([Random(2, 5, 1)] int teachersInWorkshop)
+    {
+        // Arrange
+        var id = new Guid("ca2cc30c-419c-4b00-a344-b23f0cbf18d8");
+        var changedFirstEntity = WithWorkshop(id);
+        var teachers = TeachersGenerator.Generate(teachersInWorkshop).WithWorkshop(changedFirstEntity);
+        var provider = ProvidersGenerator.Generate();
+        changedFirstEntity.Teachers = teachers;
+        changedFirstEntity.DateTimeRanges = new List<DateTimeRange>();
+        changedFirstEntity.Provider = provider;
+        SetupUpdate(changedFirstEntity);
+        var expectedTeachers = teachers.Select(s => mapper.Map<TeacherDTO>(s));
+
+        // Act
+        var result = await workshopService.Update(mapper.Map<WorkshopBaseDto>(changedFirstEntity)).ConfigureAwait(false);
+
+        // Assert
+        result.Teachers.Should().BeEquivalentTo(expectedTeachers);
+        result.AvailableSeats.Should().Be(uint.MaxValue);
```

Comment:
(Tets) Please add same `AvailableSeats` assertion to `Create()` method tests, as you've changed that service method too

Reply:
Thanks for changing, but please mind test coverage. Seems more test should be written

Reply:
I don't really get what I have to do. Should I add more test cases?